### PR TITLE
Po: update sr_RS translation

### PR DESCRIPTION
**Pull request by kostich**
Updated the Serbian translation.

### CODE REVIEW

File: po/sr.po [hunk 1]
```diff
@@ -63,119 +66,381 @@ msgstr ""
...
-#: ../data/gcolor3.appdata.xml.in.h:7
+#: data/nl.hjdskes.gcolor3.appdata.xml.in:37
 msgid "The list of saved colors"
 msgstr "Списак сачуваних боја"
 
-#: ../data/menus.ui.h:1
+#. Translators: please do NOT translate this.
+#: data/nl.hjdskes.gcolor3.appdata.xml.in:44
+msgid "Jente Hidskes"
+msgstr "Jente Hidskes"
```
Comment (Hjdskes) [inline po/sr.po]: You can leave this untranslated.

File: po/sr.po [hunk 2]
```diff
@@ -63,119 +66,381 @@ msgstr ""
...
+msgstr "Обриши означену боју"
+
+#: data/window.ui:26
 msgid "Color name..."
 msgstr "Име боје..."
 
-#: ../src/main.c:45
+#: data/window.ui:89
+msgid "page0"
+msgstr "страница0"
```
Comment (Hjdskes) [inline po/sr.po]: You can leave this untranslated.

File: po/sr.po [hunk 3]
```diff
@@ -63,119 +66,381 @@ msgstr ""
...
+
+#: data/window.ui:155
+msgid "page1"
+msgstr "страница1"
```
Comment (Hjdskes) [inline po/sr.po]: You can leave this untranslated.

File: po/sr.po [hunk 4]
```diff
@@ -63,119 +66,381 @@ msgstr ""
...
-msgid "The model is empty; there are no colors to remove"
-msgstr "Модел је празан, нема боја које се могу уклонити"
+#: src/gcolor3-window.c:397
+msgid "Picker"
+msgstr "Бирач"
 
-#: ../src/gcolor3-window.c:378
-msgid "Cannot update color: invalid path"
-msgstr "Не могу да ажурирам боју: неисправна путања"
+#~ msgid "Error opening file: %s. Colors likely won't be saved"
```
Comment (Hjdskes) [inline po/sr.po]: You can remove this line and all of the following ones.

**Comment by kostich** [comment]
Should be fine now yet GH is showing me that  there are conflicts again. What am I missing?

**Comment by Hjdskes** [comment]
Your translation commits are based on an outdated branch. Run `git fetch upstream` and then rebase your commit on top of the latest master branch using `git rebase master`. You'll get a merge conflict in the header of your translation; just take the version from current master and resolve the conflict. Then force push (`git push -f`) and it should be fixed.

**Comment by Hjdskes** [comment]
I have fixed the merge conflict locally. Your updates will be part of 2.3. Thanks!

**Comment by kostich** [comment]
@Hjdskes, thanks for sorting this out! I was quite busy recently so I really appreciate this. :)

**Comment by Hjdskes** [comment]
No problem!

**Pull request closed without merging**